diff --git a/src/pymetadata/__init__.py b/src/pymetadata/__init__.py
index 3b70f97..cf37814 100644
--- a/src/pymetadata/__init__.py
+++ b/src/pymetadata/__init__.py
@@ -7,7 +7,7 @@ __version__ = "0.4.2"


 program_name: str = "pymetadata"
-RESOURCES_DIR: Path = Path(__file__).parent / "resources"
+RESOURCES_DIR: Path = Path("/tmp/pymetadata") / "resources"
 ENUM_DIR: Path = Path(__file__).parent / "metadata"

 CACHE_USE: bool = False
